lib/floe/workflow/state.rb

```diff
@@ -101,6 +102,10 @@ def wait_until
         context.state["WaitUntil"] && Time.parse(context.state["WaitUntil"])
       end
 
+      def long_name
+        "#{self.class.name.split("::").last}:#{name}"
```

Comment:
Just for my clarity...previously this used `context.state_name` and now it's `name` - are those the same?

Reply:
These are the same, `context.state_name` is used to load the "current_state" from the `Floe::Workflow` context, once we're in a `Floe::Workflow::State` context name is a cleaner way to get at this.  (would have to walk the history but this logging might have moved out of Workflow and into Workflow::State)

Reply:
```
states = context.children.map {|name, type, args| type.new(:name => name) ... }

context.state_name = current_state_name
current_state = states.detect {|state| state.name == context.state_name}

so state.name (where state == current_state_name) == context.state_name
```